Reference/Searching/Examine/pdf-index.md

```diff
@@ -0,0 +1,85 @@
+---
+versionFrom: 8.0.0
+---
+
+# Pdf indexes and multisearchers
+
+If you want to index pdfs and search for them you will need to use the [UmbracoExamine.Pdf extension package](https://github.com/umbraco/UmbracoExamine.PDF).
+
+## Installation
+
+Install with Nuget:
+`Install-Package UmbracoCms.UmbracoExamine.PDF`
+
+You will then have a new Examine index called "PDFIndex" available.
```

Comment:
_Examine index_ - really a good one for our style guide.
I think it should be _Examine Index_, but perhaps that's wrong? 🤔

Reply:
think that is wrong, an index is just a term, like you would say file - not File 🙂